pkg/foundation/metrics/measure/measure.go

```diff
@@ -28,6 +28,16 @@ var (
 	PipelinesGauge = metrics.NewLabeledGauge("conduit_pipelines",
 		"Number of pipelines by status.",
 		[]string{"status"})
+	PipelineStatusGauge = metrics.NewLabeledGauge(
```

Comment:
Note: I considered another approach, where the value would be the integer value of status itself, e.g. `conduit_pipeline_status{pipeline_name="testpipeline1"} 3` (where for example 3=degraded). While that's a bit more readable and meaningful (reads as "the status of pipeline testpipeline1 is '3') than `conduit_pipeline_status{pipeline_name="pipeline1",status="degraded"} 1` (the gauge value '1' has no meaning), the integer value spoils it all.:)

Reply:
Hmm, interesting, that could work if we describe the meaning of the integers in the description of the metric. The whole problem is that Prometheus doesn't allow string values for metrics, so anything we do here is a workaround, but yours might be easier to handle when visualizing the metrics in Grafana or other tools.

Reply:
But it would still be numbers when visualizing, right? Unless some custom mapping is done. And we would need to be careful when adding new statuses and/or refactoring.

Reply:
I looked a bit into how the data can be used within Grafana, for visualizations and alerting. For visualization, I built a simple table dashboard showing a pipeline's current and/or historical statuses.
* Option with the metric value representing the integer status (e.g. `conduit_pipeline_status{pipeline_name="testpipeline1"} 3` (where for example 3=degraded)
  * dashboard: needs value mappings, that can be done with field overrides (5 of them)
  * alert: same, the alert checks if the value of `conduit_pipeline_status{pipeline_name="testpipeline1"}`
* Option with the status being set as a label name (e.g. `conduit_pipeline_status{pipeline_name="pipeline1",status="degraded"} 1`)
  * dashboard: nothing special needs to be done. However, other visualizations (such as charts) could be more difficult to build, because the actual status needs to be taken from a label.
  * alert: the alert checks if the value for `conduit_pipeline_status{pipeline_name="pipeline1",status="degraded"}` is above 0. This is counter-intuitive since the actual value plays no role (we're checking for the mere presence of the metric).

I'm still a bit on the fence, because of the value mappings for option 1, but overall it _feels_ like it's more in the spirit of Prometheus.

Reply:
We discussed this on today's standup, we're going with option 1 (the gauge value is the pipeline's status as integer).